### PR TITLE
CA-334951: Ignore 0-star recommendations from WLB (backport)

### PR DESCRIPTION
**Pull request by psafont**
This is a backport of https://github.com/xapi-project/xen-api/pull/4030, minus the cleanups done after the actual fixes.

### CODE REVIEW

File: ocaml/xapi/xapi_vm.ml [hunk 1]
```diff
@@ -61,8 +61,16 @@ let assert_can_boot_here ~__context ~self ~host =
   assert_can_boot_here ~__context ~self ~host ~snapshot ()
 
 let retrieve_wlb_recommendations ~__context ~vm =
+  let open Workload_balancing in
```
Comment (lippirk) [inline ocaml/xapi/xapi_vm.ml]: why did you add this `open`?

Comment (psafont) [reply ocaml/xapi/xapi_vm.ml]: The syntax `Module.(...)` is not supported in OCaml 4.02.3, so I used a local open instead to avoid repetition.

**Comment by lindig** [comment]
@lippirk have your comments been addressed such that we could merge this?
